cmake/Modules/Packages/MLIAP.cmake

```diff
@@ -0,0 +1,29 @@
+# if PYTHON package is included we may also include Python support in MLIAP
+set(MLIAP_ENABLE_PYTHON_DEFAULT OFF)
+if(PKG_PYTHON)
+  find_package(Cythonize)
+  if(Cythonize_FOUND)
+    set(MLIAP_ENABLE_PYTHON_DEFAULT ON)
+  endif()
+endif()
+
+option(MLIAP_ENABLE_PYTHON "Build MLIAP package with Python support" ${MLIAP_ENABLE_PYTHON_DEFAULT})
+
+if(MLIAP_ENABLE_PYTHON)
+  find_package(Cythonize REQUIRED)
+  if(NOT PKG_PYTHON)
+    message(FATAL_ERROR "Must install PYTHON package for MLIAP with Python support")
+  endif()
+
+  set(MLIAP_CYTHON_DIR ${CMAKE_BINARY_DIR}/cython)
+  file(MAKE_DIRECTORY ${MLIAP_CYTHON_DIR})
+  add_custom_command(OUTPUT  ${MLIAP_CYTHON_DIR}/mliap_model_python_couple.cpp ${MLIAP_CYTHON_DIR}/mliap_model_python_couple.h
+            COMMAND            ${CMAKE_COMMAND} -E copy ${LAMMPS_SOURCE_DIR}/MLIAP/mliap_model_python_couple.pyx ${MLIAP_CYTHON_DIR}/mliap_model_python_couple.pyx
```

Comment:
Do you really have to copy it? Just refer to it using the full path.

Reply:
Yes, this is required. `cythonize` will create the source code at the location of the the .pyx file and we don't want created files mixed with the source.

Reply:
Thanks,

Reply:
we could use `copy_if_different` instead of `copy` here.